### PR TITLE
Add React-specific content for perf-monitoring.

### PR DESCRIPTION
**Pull request by onurtemizkan**
Resolves: #3216

### CODE REVIEW

**Comment by imatwawana** [comment]
All of the includes that you're replacing have the following sentence at the beginning of them that needs to be updated so that it no longer references `@sentry/react`:

// If you're using one of our integration packages, like `@sentry/react` or `@sentry/angular`,
// substitute its name for `@sentry/browser` here

Otherwise this LGTM, but leaving final approval to @PeloWriter

**Comment by onurtemizkan** [comment]
Thanks, @imatwawana. Removed the references from the common content.

File: src/includes/performance/enable-automatic-instrumentation/javascript.react.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }),
+  ],
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0,
+});
+```
+
+Now you should be generating `pageload`/`navigation` transactions from the `BrowserTracing` integration, using `react-router` instrumentation.
```
Comment (AbhiPrasad) [inline src/includes/performance/enable-automatic-instrumentation/javascript.react.mdx:34]:
```suggestion
Now you should be generating `pageload`/`navigation` transactions from the `BrowserTracing` integration, using Sentry's [`react-router` instrumentation](/platforms/javascript/guides/react/configuration/integrations/react-router/).
```

**Review by AbhiPrasad — APPROVED** [review]

**Comment by imatwawana** [comment]
@onurtemizkan - This is fine from the docs side. Shall I merge this?

**Comment by onurtemizkan** [comment]
@imatwawana - Yes please, thanks.

**Pull request merged**